### PR TITLE
derivative fuzzy digamma

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: include/eve/function/ellint_rd.hpp [hunk 1]
```diff
@@ -25,26 +25,29 @@ namespace eve
...
+  //!`x`, `y`, `z`:   [floating real values](@ref eve::floating_real_value).
   //!
   //!  This computes the Carlson's elliptic integral
   //!
   //!  \f[ R_D(x, y, z) = \frac32 \int_{0}^{\infty} [(t+x)(t+y)]^{-1/2} (t+z)^{-3/2} \mbox{d}t\f]
   //!
   //!  as described in  Carlson, Numerische Mathematik, vol 33, 1 (1979)
   //!
-  //!  Parameters `x` and `y`must be non-negative and at most one ot them  zero. Parameter `z` must be greater than 0.
+  //!  Parameters `x` and `y`must be non-negative and at most one ot them  zero.
```
Comment (jfalcou) [inline include/eve/function/ellint_rd.hpp:49]: This should be in the parameters description directly

File: include/eve/function/expx2.hpp [hunk 2]
```diff
@@ -12,6 +12,68 @@
...
+  //!
+  //! @brief Callable object computing \f$e^{\pm x^2}\f$.
+  //!
+  //! **Required header:** `#include <eve/function/expx2.hpp>`
+  //!
+  //! #### Members Functions
+  //!
+  //! | Member       | Effect                                                     |
+  //! |:-------------|:-----------------------------------------------------------|
+  //! | `operator()` | the  computation of expx2                                  |
```
Comment (jfalcou) [inline include/eve/function/expx2.hpp:28]: This should be a sentence with a verb no ?

File: include/eve/function/fuzzy.hpp [hunk 3]
```diff
@@ -61,7 +61,7 @@ namespace eve
   //! @{
   //! @var almost
   //!
-  //! @brief  Higher-order @callable imbuing more standard semantic onto other @callable{s}.
+  //! @brief  Higher-order @callable imbuing a toerant to little errors semantic onto other @callable{s}.
```
Comment (jfalcou) [inline include/eve/function/fuzzy.hpp:64]: tolerant to small errors

File: include/eve/module/real/math/function/regular/generic/exp2.hpp [hunk 4]
```diff
@@ -41,6 +41,7 @@
 #include <eve/function/sqr.hpp>
 #include <eve/module/real/core/detail/generic/horn.hpp>
 #include <type_traits>
+#include <iostream>
```
Comment (jfalcou) [inline include/eve/module/real/math/function/regular/generic/exp2.hpp:44]: Should be removed

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: include/eve/function/expx2.hpp (hunk 2 above)
Comment (jtlap) [reply include/eve/function/expx2.hpp:28]: done

File: include/eve/module/real/math/function/regular/generic/exp2.hpp (hunk 4 above)
Comment (jtlap) [reply include/eve/module/real/math/function/regular/generic/exp2.hpp:44]: done

File: include/eve/function/expx2.hpp (hunk 2 above)
Comment (jtlap) [reply include/eve/function/expx2.hpp]: done

File: include/eve/function/fuzzy.hpp (hunk 3 above)
Comment (jtlap) [reply include/eve/function/fuzzy.hpp:64]: done

Comment (jtlap) [reply include/eve/function/fuzzy.hpp:64]: done

File: include/eve/module/real/math/function/regular/generic/exp2.hpp (hunk 4 above)
Comment (jtlap) [reply include/eve/module/real/math/function/regular/generic/exp2.hpp]: done

File: include/eve/function/fuzzy.hpp (hunk 3 above)
Comment (jfalcou) [reply include/eve/function/fuzzy.hpp:64]: this one is still wrong

Comment (jtlap) [reply include/eve/function/fuzzy.hpp:64]: done

**Review by jfalcou — APPROVED** [review]

**Pull request merged**